.github/workflows/push.yml

```
@@ -1,16 +1,27 @@
 name: GitHub Actions
 on: [ push ]
+concurrency: 'ci-${{ github.ref }}'
 env:
   APP_IMAGE: "encryption-api-php-client"
   DOCKERHUB_USER: "keboolabot"
   DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
 jobs:
-  Build:
+  build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: ['7.4', '8.2']
     steps:
       - name: Check out the repo
         uses: actions/checkout@v4
-      - name: Build image and run tests
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ env.DOCKERHUB_USER }}
+          password: ${{ env.DOCKERHUB_TOKEN }}
+
+      - name: Build and run tests
         run: |
-          docker login --username "$DOCKERHUB_USER" --password "$DOCKERHUB_TOKEN"
-          docker compose run ci
+          docker compose build --build-arg PHP_VERSION=${{ matrix.php-version }}
```

Neni potreba buildit vsechny services
```suggestion
          docker compose build ci --build-arg PHP_VERSION=${{ matrix.php-version }}
```